csharp/ql/test/utils/modelgenerator/dataflow/Sinks.cs

```diff
@@ -20,6 +20,10 @@ public class NewSinks
     // neutral=Sinks;NewSinks;Sink2;(System.Object);summary;df-generated
     public static void Sink2(object o) => throw null;
 
+    // Defined as sink neutral in the file next to the neutral summary test.
+    // neutral=Sinks;NewSinks;NoSink;(System.Object);summary;df-generated
+    public static void NoSink(object o) => throw null;
```

Comment:
Now the bug is fixed, isn't this a confusing name? (Same for the java test.)

Reply:
The method is defined as sink (manual) and summary (df-generated) neutral in the extensible file.
That is, we expect that the "neutral" test (which only reports summary neutrals) would generate a neutral (df-generated) summary for this method.
The name "NoSink" is because it is explicitly stated in the extensible file that this method is sink neutral (it is not a sink).